reverse_engineering/helpers/clusterHelper.js

```diff
@@ -401,21 +401,21 @@ const getDbCollectionData = async ({
 	});
 	const documentKind = data.documentKinds?.[bucketName]?.documentKindName || '';
```

Comment:
Are the document kinds still used?

Reply:
Document kinds will be removed in the next task